defaults/main.yml

```diff
@@ -44,13 +44,16 @@ prometheus_targets: []
 #    env: test
 #    job: node
 
-prometheus_scrape_jobs: []
-#prometheus_scrape_jobs:
-#- job_name: "node"
-#  metrics_path: /metrics
-#  static_configs:
-#  - targets:
-#    - "demo.cloudalchemy.org:9100"
+prometheus_scrape_configs:
+- job_name: "prometheus"
+  metrics_path: {{ prometheus_metrics_path }}
```

Comment:
Ansible wants to have jinja variables enclosed in quote marks.
http://docs.ansible.com/ansible/latest/YAMLSyntax.html#gotchas